Library/Homebrew/description_cache_store.rb

```diff
@@ -60,7 +63,10 @@ def update_from_report!(report)
   # @param formula_names [Array] the formulae to update
   # @return [nil]
   def update_from_formula_names!(formula_names)
-    return unless Homebrew::EnvConfig.eval_all?
+    unless Homebrew::EnvConfig.eval_all?
+      database.clear
```

Comment:
```suggestion
      database.clear!
```

Reply:
Funny enough I actually had the ! in my first revision of this but removed it to match `CacheStoreDatabase.delete` which also lacks a !.

I'll add this back.